yaml/csr.yaml.m4

```diff
@@ -2730,6 +2741,7 @@ ifelse(eval(UMODE != 0), 1, [[[
       lsb: 15
       warl_legalize: |
         val_out = 0
+]]])
```

Comment:
Same here. This clause is okay, but the next one is wrong. If P_EXT != 0, then the P bit will be hardwired to 1 (so WARL (0x1))